x-pack/plugins/security_solution/scripts/run_cypress/utils.ts

```diff
@@ -156,3 +158,22 @@ export const getOnBeforeHook = (module: unknown, beforeSpecFilePath: string): Fu
 
   return module.onBeforeHook;
 };
+
+/**
+ * Sets the default log level for `ToolingLog` instances created by `createToolingLogger()`:
+ * `x-pack/plugins/security_solution/common/endpoint/data_loaders/utils.ts:148`.
+ * It will first check the NodeJs `process.env` to see if an Environment Variable was set
+ * and then, if provided, it will use the value defined in the Cypress Config. file.
+ */
+export const setDefaultToolingLoggingLevel = (defaultFallbackLoggingLevel?: string) => {
+  const logLevel =
+    process.env.TOOLING_LOG_LEVEL ||
+    process.env.CYPRESS_TOOLING_LOG_LEVEL ||
+    defaultFallbackLoggingLevel ||
+    '';
```

Comment:
nit: It looks like empty string isn't necessary here since if condition below is gonna check for all "falsable" values.